### PR TITLE
-Desktop: Resolves issue #4096: disabled the auto update option in linux

### PR DESCRIPTION
**Pull request by anshuman9999**

### CODE REVIEW

**Comment by laurent22** [comment]
Sorry it's not what was asked and so few changes there are several problems down to the details, like "linux" instead of "Linux".

**Comment by anshuman9999** [comment]
> Sorry it's not what was asked and so few changes there are several problems down to the details, like "linux" instead of "Linux"
Should I open the pull request again after fixing these issues

**Comment by anshuman9999** [comment]
> Sorry it's not what was asked and so few changes there are several problems down to the details, like "linux" instead of "Linux".

Can you please tell me what string do I have to change and can I create another PR after fixing them.
Thanks

**Comment by CalebJohn** [comment]
@laurent22 I had asked for the [change in the public field](https://github.com/laurent22/joplin/pulls#issuecomment-729400930) To me it makes more sense to completely hide the field rather than display a warning (since it's a non-functioning button on linux).

**Comment by laurent22** [comment]
Ah yes, sorry I missed that. And also I've just realised that the change to "Note: Does not work in Linux" as I've suggested is not right, since it works in certain Linux desktop environments, thus the original "Note: Does not work in all desktop environments.".

Let's just only hide the "autoUpdateEnabled" setting on Linux then, and I hope there won't be any subtle side effect from changing the setting from public to private.

**Comment by CalebJohn** [comment]
@anshuman9999 Sorry for the confusion, but to finish this pull request please add a new commit that removes the change to line 601. This will be good after that!

**Comment by anshuman9999** [comment]
@CalebJohn I accidently commited this above commit. Should I close this PR and make a new one?

**Comment by CalebJohn** [comment]
@anshuman9999 no need, simply remove the mistaken commit and force push when you have a new one. Or just leave it, doesn't look like it's doing anything and all commits will be squashed when the PR is accepted.

**Comment by anshuman9999** [comment]
@CalebJohn Thank you so much for helping me out. I have commited the changes. Please review it.

File: packages/lib/models/Setting.ts [hunk 1]
```diff
@@ -602,7 +602,7 @@ class Setting extends BaseModel {
 				},
 			},
 
-			startMinimized: { value: false, type: SettingItemType.Bool, section: 'application', public: true, appTypes: ['desktop'], label: () => _('Start application minimised in the tray icon') },
+			startMinimized: { value: false, type: SettingItemType.Bool, section: 'application', public: true, appTypes: ['desktop'], label: () => _('Start application minimised in the tray icon testing...') },
```
Comment (CalebJohn) [inline packages/lib/models/Setting.ts:605]: Looks like you didn't mean to change this...

Comment (anshuman9999) [reply packages/lib/models/Setting.ts:605]: I'm so sorry. It was meant to be for my testing and I pushed it accidently. I am fixing it asap

Comment (anshuman9999) [reply packages/lib/models/Setting.ts]: I have pushed the changes again. Please review it.

**Comment by CalebJohn** [comment]
@anshuman9999 Looks Good! Thanks for the fix!
@laurent22 I'll leave the merge to you

**Comment by laurent22** [comment]
Looks good, thanks for the update.

**Comment by anshuman9999** [comment]
Thank you all so much for helping me out. I am a beginner and this meant a lot!

**Comment by anshuman9999** [comment]
@CalebJohn  @laurent22  Thank you  much for helping me out in this!

**Comment by anshuman9999** [comment]
@laurent22 @CalebJohn  Thank you so much for helping me out!

**Pull request merged**